Tests/RoslynPluginGeneratorTests/NuGetPackageHandlerTests.cs

```diff
@@ -36,80 +36,77 @@ public class NuGetPackageHandlerTests
         public void NuGet_TestPackageDownload_Release_Release()
         {
             // Arrange
-            string testDir = TestUtils.CreateTestDirectory(this.TestContext);
-            string testDownloadDir = Path.Combine(testDir, "download");
+            string targetNuGetRoot = TestUtils.CreateTestDirectory(this.TestContext, ".nuget.target");
 
             // Create test NuGet payload and packages
-            BuildTestPackages(true, true);
```

Comment:
I modified the _BuildTestPackages_ method to create and return the repo to use. This simplifies the calling code.